### ISSUE TITLE
Unicode error halts, but renamed file appears run directory

### ISSUE THREAD

**Issue by haggletonpie** [jkwill87/mnamer#73]
I ran...

`mnamer -r .`

... and get this error, but the file does appear renamed, but moved to the directory the script ran in.

... instead of renaming the file in place.

This was my first run, so I haven't yet explored the options, especially since it quits.

Cool project tho.

```
Traceback (most recent call last):
  File "/usr/local/bin/mnamer", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.5/dist-packages/mnamer/__main__.py", line 385, in main
    process_files(targets, user_media, test_run, id_key, **config)
  File "/usr/local/lib/python3.5/dist-packages/mnamer/__main__.py", line 302, in process_files
    notify.info("Relocating file to '%s'" % dest_path, True)
  File "/usr/local/lib/python3.5/dist-packages/mnamer/__main__.py", line 63, in info
    print(self.bullet_text + text if bullet else text)
  File "/usr/local/lib/python3.5/dist-packages/colorama/ansitowin32.py", line 41, in write
    self.__convertor.write(text)
  File "/usr/local/lib/python3.5/dist-packages/colorama/ansitowin32.py", line 164, in write
    self.wrapped.write(text)
UnicodeEncodeError: 'latin-1' codec can't encode character '\u2022' in position 2: ordinal not in range(256)
```

**Comment by jkwill87** [comment]
Hey, thanks for trying mnamer. A couple of things I notice here--

1. You are running Python 3.5 but mnamer only support Python 3.6+.
2. This exception seems to be attributable to a conflict with `colorama`, a package which is unrelated to mnamer (at this time of writing mnamer is only dependant on appdirs, babelfish, certifi, chardet, guessit, idna, python-dateutil, rebulk, requests, requests-cache, six, and teletype).

This should work for you if you update Python and [install mnamer in an isolated environment](https://github.com/jkwill87/mnamer/wiki/Installation#installing-using-pipx).

**Comment by haggletonpie** [comment]
Ugh. Thx for releasing this. Seemed/seems like a cool project, wanted to try it out, spread the word. 

Too much install trouble for me on Ubuntu 16.04. Some feedback...

-- Python 3.6 requirement is apparently critical, and not mentioned, or at least I couldn't find it. I get it: it's 3 years old, but there's still a lot of 16.04 out there... and that's 3.5 'baked in'.

-- after upgrading to 3.6, and editing the mnamer script to use python3.6 explicitly (not just python3, which is 3.5 on ubuntu 16.04), the colorama error returns. Confirmed on two terminals, Putty on win, and ubuntu terminal.

-- installing pipx was not successful.

-- bottomline: I tried installing on two 16.04 seedboxes, and it went poorly, ultimately failing for lack of upgrade troubleshooting time. It might be good to explain that the mnamer, such as released, needs a system that has been fully upgraded (including pip, pip3, python3, python3.6, pipx... all that) to 3.6. I just don't have the time to do all that just now, just to try out a neat project.

That's it for me. Thx for makin' code and puttin' it out there.

**Comment by jkwill87** [comment]
Took another look. setup.py actually had 3.7 as the minimum version so installing under 3.6 would be installing mnamer v1, an older version of the program. I've bumped this down to 3.6 and [it seems to be working now](https://travis-ci.org/github/jkwill87/mnamer/jobs/677793933). I pushed this update to PyPI as v2.3.3 so if you try again under Python 3.6 hopefully it should work for you. mnamer currently uses Python 3.6 features like variable annotations which have not been backported to 3.5 so unfortunately that particular version will not be supportable in v2.

Thanks again for bringing this up!
